# Module-6-Challenge
in vacationpy instead of city_data_df = pd.read_csv("output_data/cities.csv") the path was city_data_df = pd.read_csv("cities.csv"). I ended up moving the file because for some reason I was having a pathing issue. I also did not create the map in vacationpy
